### PR TITLE
[ICA] -  Re-usable Resources - 6044, 6045, 6046, 6047, 6048 submitted

### PR DESCRIPTION
**Pull request by seanmcilroy29**
RR submission from ICA - See #199 
RR 6044, 6045, 6046, 6047, 6048

Validated with no errors

### CODE REVIEW

**Comment by jpradocueva** [comment]
@seanmcilroy29 lets discuss this PR. I have a different views about use the prefix [ADMIN]; we are suggesting new Objects. This need WG approval.

**Comment by seanmcilroy29** [comment]
@jpradocueva -  I was going to request the WG review this PR after the submission from ICA.
 [ADMIN] would have been removed at this point.
To make it easier for the WG, I'll break the PR into two

File: Common.xml [hunk 1]
```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
...
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Name>Battery Percentage</Name>     
+        <Operations>R</Operations>     
+        <Type>Float</Type>
+        <RangeEnumeration>0.00-100.00</RangeEnumeration>
+        <Units>%</Units>
```
Comment (akeranen) [inline Common.xml]: Currently “%” is for values from 0 to 1. I’d recommend leaving this empty for now until we register a proper unit for percentage 0 to 100.

File: Common.xml [hunk 2]
```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
...
       
       <Item ID="6046">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
+        <Name>Smoke Sensor State</Name>     
+        <Operations>R</Operations>     
+        <Type>Boolean</Type>
+        <RangeEnumeration>Normal (0) or Smoke detected (1)</RangeEnumeration>
```
Comment (akeranen) [inline Common.xml]: RangeEnumeration field should be machine-readable. See [Appendix D of LwM2M TS 1.1.1](http://www.openmobilealliance.org/release/LightweightM2M/V1_1_1-20190617-A/OMA-TS-LightweightM2M_Core-V1_1_1-20190617-A.pdf). The explanation of values should be in description.

That said, for boolean values there’s only “true” and “false” value possible so RangeEnumeration does not provide much value here and can be left empty.

File: Common.xml [hunk 3]
```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
...
       
       <Item ID="6047">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Name>CO2</Name>
```
Comment (akeranen) [inline Common.xml]: This name should be more descriptive (e.g., “Detected CO2 percentage”?)

File: Common.xml [hunk 4]
```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
...
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Name>CO2</Name>     
+        <Operations>R</Operations>     
+        <Type>Float</Type>
+        <RangeEnumeration>0.00-100.00</RangeEnumeration>
+        <Units>%</Units>
+        <Submitter>IoT Connectivity Alliance</Submitter>
+        <Description>Current detected CO2 percentage</Description>
```
Comment (akeranen) [inline Common.xml]: Could be clarified as “Current level of detected CO2 as percentage (of what?”)

File: Common.xml [hunk 5]
```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
...
         <TS></TS>
         <TSLink>0</TSLink>
       </Item>
       
       <Item ID="6048">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
+        <Name>Decibel Value</Name>
```
Comment (akeranen) [inline Common.xml]: For example “Alarm loudness” sounds like more descriptive name for this.

**Review by akeranen — CHANGES_REQUESTED** [review]

File: Common.xml (hunk 1 above)
Comment (QingAn) [reply Common.xml]: Agree

File: Common.xml (hunk 2 above)
Comment (QingAn) [reply Common.xml]: What about change the Type to Enumeration?

File: Common.xml (hunk 3 above)
Comment (QingAn) [reply Common.xml]: I can live with “Detected CO2 percentage”

File: Common.xml (hunk 4 above)
Comment (QingAn) [reply Common.xml]: Yes, it could. Should I update or you can edit directly?

File: Common.xml (hunk 5 above)
Comment (QingAn) [reply Common.xml]: I can live with that. Should I update or you can edit directly?

File: Common.xml (hunk 4 above)
Comment (seanmcilroy29) [reply Common.xml]: @QingAn - Could you clarify `of 'what'` this is a percentage of, as suggested by Ari and I'll apply the update to the description.

File: Common.xml (hunk 2 above)
Comment (seanmcilroy29) [reply Common.xml]: @QingAn - Your suggestion to change the Type to `Enumeration` can't be applied as this isn't a `Data Type` - [View Values](http://www.openmobilealliance.org/release/LightweightM2M/V1_1_1-20190617-A/OMA-TS-LightweightM2M_Core-V1_1_1-20190617-A.pdf)

Did you want to update the `RangeEnumeration` to blank as suggested by Ari?

Comment (QingAn) [reply Common.xml]: @seanmcilroy29 Ok. Blank can work for me.

File: Common.xml (hunk 4 above)
Comment (QingAn) [reply Common.xml]: @seanmcilroy29 Yes, please apply the suggestion made by @akeranen . Thanks

**Comment by QingAn** [comment]
Hi, are all the comment resolved? May I ask what would be the next step?
Thanks

**Comment by seanmcilroy29** [comment]
@QingAn - The IPSO Working Group (WG) convenes on the 5th Sept after the summer break.
During the WG Conference Call on the 5th, your submission will be reviewed

File: Common.xml [hunk 6]
```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
...
       
       <Item ID="6044">    
-        <Name>Reserved</Name>     
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
+        <Name>Battery Percentage</Name>     
+        <Operations>R</Operations>     
+        <Type>Float</Type>
+        <RangeEnumeration>0.00-100.00</RangeEnumeration>
```
Comment (mcocak) [inline Common.xml]: Should be "0.00..100.00"

File: Common.xml [hunk 7]
```diff
@@ -3459,61 +3459,61 @@ Using values in range 0-127 is recommended to avoid ambiguities with byte order
...
-        <Operations></Operations>     
-        <Type></Type>
-        <RangeEnumeration></RangeEnumeration>
-        <Units></Units>
-        <Submitter></Submitter>
-        <Description></Description>
+        <Name>Detected CO2 percentage</Name>     
+        <Operations>R</Operations>     
+        <Type>Float</Type>
+        <RangeEnumeration>0.00-100.00</RangeEnumeration>
```
Comment (mcocak) [inline Common.xml]: should be "0.00..100.00"

**Comment by mkgillmore** [comment]
@jpradocueva - group conditionally agrees to merge 10/9/2019 pending conflict resolution and a comment about about range enumeration with 6044, and 6047 not using correct format.

**Pull request merged**